recipes/ablang2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "ablang2" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ablang2-{{ version }}.tar.gz
+  sha256: 7f01bec5c5a5d5c270fa078cdd923d3a98bdc2b93538b17d46231db2d783676d
+
+build:
+  skip: True  # [win]
```

Comment:
It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.